acme_diags/driver/enso_diags_driver.py

```diff
@@ -54,15 +54,26 @@ def calculate_nino_index_model(data, nino_region_str, parameter):
     """
 
     try:
-        sst = data.get_timeseries_variable('SST')
+        try:
+            # Try sea surface temperature first.
+            sst = data.get_timeseries_variable('SST')
```

Comment:
@chengzhuzhang I thought it would be good to at least try `SST` first and otherwise default to `TS`. Let me know if we should always go with `TS`.

Reply:
yeah, good idea.